free-3.md

```diff
@@ -4,51 +4,96 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+A DSL to describe video game tactics. This would help newer plays learn from more
+experienced players by providing experienced players a way to share tactical or
+strategic advice.
```

Comment:
I agree, I think the video game experience is so wide-spread that it is hard to come up with consistent syntax and semantics. For example, how do I relate CoD and Solitaire?